Cargo.toml

```diff
@@ -486,8 +486,8 @@ overflow-checks = false
 rpath = false
 
 [profile.release.package.server]
-opt-level = 1
-codegen-units = 4
+opt-level = 4
+codegen-units = 1
```

Comment:
I'm curious if this change has any benefits.

Reply:
This must be a misconfiguration, I'll roll it back.